code/modules/events/spacevine.dm

```diff
@@ -112,6 +112,9 @@
 		. = 1
 		QDEL_IN(holder, 5)
 
+/datum/spacevine_mutation/aggressive_spread/proc/aggrospread_act(obj/structure/spacevine/S, mob/living/M)
+	return
+
```

Comment:
Now aggressive spreading won't work at all. This PR needs to leave the spacevine.dm file alone and keep ex_act as placeholder.